src/traces/scattergeo/plot.js

```diff
@@ -62,6 +62,7 @@ module.exports = function plot(geo, calcData) {
                 .data([{geojson: lineData, trace: trace}])
               .enter().append('path')
                 .classed('js-line', true)
+                .style('stroke-miterlimit', 2);
```

Comment:
❤️ 
I also looked around at the paths generated by geojson/d3, which do not have any explicit `stroke-miterlimit` set (so they get the default of 4) and they generally look fine, but in certain cases might also look nicer at 2.
Here's the northeast US (global scale) at default 4:
![screen shot 2017-09-22 at 4 30 22 pm](https://user-images.githubusercontent.com/2678795/30763292-86572c32-9fb3-11e7-8677-b2a06d354bb1.png)

and at 2:
![screen shot 2017-09-22 at 4 30 40 pm](https://user-images.githubusercontent.com/2678795/30763301-8bfe5a98-9fb3-11e7-9e95-09b37109f2b3.png)

Reply:
done in https://github.com/plotly/plotly.js/pull/2030/commits/303de22a6d8b8e56f82c88783469a155f4629c7b

Reply:
Sweet, thanks for the special mock to call this out - rather than relying on subpixel tweaks elsewhere to test it.